python/xorbits/_mars/services/storage/tests/test_transfer.py

```diff
@@ -155,13 +161,34 @@ async def get_receiver_ref(address: str, band_name: str):
             address=address, uid=MockReceiverManagerActor.default_uid()
         )
 
+    async def _copy_to_receiver(
+        self,
+        receiver_ref: mo.ActorRefType["ReceiverManagerActor"],
+        local_buffers: List,
+        remote_buffers: List,
+        session_id: str,
+        data_keys: List[str],
+        block_size: int,
+    ):
+        await asyncio.sleep(3)
```

Comment:
Why sleep(3)?

Reply:
> Why sleep(3)?

``test_cancel_transfer`` this UT tests the case that ``send_batch_data`` would be cancelled. To simulate this, an actor is mocked, and sleeping 3 seconds before the buffer is actually created ensures that it must be a task cancel since ``cancel`` is called 0.5 seconds after the ``send_batch_data`` function is called.